pages/Announcements/Overview.md

```diff
@@ -56,12 +56,15 @@ Announcements in [Batch Publication Files](/BatchPublications/Overview) have an
 The `DSNPBatchPublication` Ethereum events are ordered by information provided in the transaction.
 Announcements in a Batch Publication File are then ordered by row index.
 
-
 1. `DSNPBatchPublication` Block number ascending
 2. `DSNPBatchPublication` Transaction index ascending
 3. `DSNPBatchPublication` Log index ascending
 4. Batch Publication File Announcement row appearance order
 
+## Reverting an Announcement
+
+Announcements may not be deleted, but some may be marked as invalid by using a [Tombstone Announcement](/Announcements/Types/Tombstone).
+For example, if a user creates a reaction announcement, they may remove that reaction by creating a tombstone announcement.
```

Comment:
There was some ongoing discussion about this at some point. Did we decide that we would use Tombstones for Reactions?

Reply:
Same here, I'd prefer that Tombstone is a revert of an invalid announcement to begin with, where invalid means a bad signature or the like. I think clients should have to handle things like accidental follow/unfollow clicks, by throttling, setting timeouts, use of confirm dialogs, etc.  Tombstoning should have a lot of friction.

Reply:
A "invalid announcement to begin with" would not need a tombstone as it is already invalid. A tombstone invalidates a previously valid signature.

That said I agree that clients ought to do all of the mentioned things like throttling. (However we should not note that in the specification. That would be a good implementation walk through as well as good to put into the Example Client)

Reply:
> A "invalid announcement to begin with" would not need a tombstone as it is already invalid.
Right, of course. I should have said, "a retraction for something posted in error or, e.g. by a malicious person who  no longer has permissions on that account", not technically invalid.  It still should have a lot of friction.

Reply:
Ah that makes more sense.

The malicious person version should be done via delegation removal not via tombstoning however. The point of a tombstone is to say that something you did, you want to undo. I don't know of a way to increase the friction any more than just an announcement. (outside of PoW type stuff that is)